usagov/usagov-2021
# Usagov 1248 shell additions

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1248

## Description


Add nano and tome log list  helper alias.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Gain access to the cms terminal:
-  If testing in one of the cloud foundry environments use `cf ssh cms`
- If testing locally build as usual, then use `docker exec -it cms sh`
2. Load the global profile with the command `source /etc/profile`
3. The text editor `nano` should be available.
4. The command `tll` should tail the latest *.log file present in the current directory.  If no log files are present you should see a usage message.

### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
